go/pkg/log/sysdb/sysdb.go

```diff
@@ -0,0 +1,52 @@
+package sysdb
+
+import (
+	"context"
+	"log"
+
+	"github.com/chroma-core/chroma/go/pkg/proto/coordinatorpb"
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/credentials/insecure"
+)
+
+type ISysDB interface {
+	CheckCollection(ctx context.Context, collectionId string) (bool, error)
+	AddCollection(ctx context.Context, collectionId string) error
+}
+
+type SysDB struct {
+	client coordinatorpb.SysDBClient
+}
+
+func NewSysDB(conn string) *SysDB {
+	grpcClient, err := grpc.Dial(conn, grpc.WithTransportCredentials(insecure.NewCredentials()))
```

Comment:
I think we should configure retries etc, fine with this being a TODO but can we track it